File: Java/프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        int[] answer = new int[numbers.length*(numbers.length-1)];
        int temp = 0;
        for(int i=0; i<numbers.length; i++) {
            for(int j = 0; j<numbers.length; j++) {
                if(i == j) continue;
                answer[temp] = numbers[i]+numbers[j];
                temp++;
            }
        }
        Arrays.sort(answer);
        return Arrays.stream(answer).distinct().toArray();
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.kt
class Solution {
    fun solution(numbers: IntArray): IntArray {
        var answer: IntArray = intArrayOf()
        val a = mutableListOf<Int>()
        
        for(i in numbers.indices) { 
            for(j in numbers.indices) {
                if(i != j) {
                  a.add(numbers[i]+numbers[j])  
                }
            } 
        }
        a.sort()
        answer = a.distinct().toIntArray()
        return answer
    }
}